packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -73,29 +82,61 @@ export class AzureClient {
             config: {},
         });
 
-        const rootDataObject = await requestFluidObject<RootDataObject>(
+        const fluidContainer = await this.createFluidContainer(
             container,
-            "/",
+            this.props.connection,
         );
-        const createNewRequest = createAzureCreateNewRequest(
-            this.props.connection.orderer,
-            this.props.connection.storage,
-            this.props.connection.tenantId,
+        const services = this.getContainerServices(container);
+        return { container: fluidContainer, services };
+    }
+
+    /**
+     * Recreates new container out of specific version of another container
+     * @param id - Unique ID of the source container in Azure Fluid Relay.
+     * @param version - Unique version of the source container in Azure Fluid Relay.
+     * @param containerSchema - Container schema used to access data objects in the container.
+     * @returns New container instance along with associated services.
+     */
+    public async reCreateContainer(
+        id: string,
+        version: string,
```

Comment:
We could make this optional and default to "latest"

Reply:
defaulting to "latest" seems like good design here

Reply:
What happens when you create the latest here?  Does it become equivalent to a call to `getContainer` for the same arguments without the trailing ops?

Reply:
I would assume new container id since this will generate a new container in AFR

Reply:
Yes - this call lands you a new container ID. That's the main reason behind keeping this API separate from `getContainer`.

Reply:
Instead of `string` we are now using `AzureContainerVersion`. It may give us more flexibility down the line if we switch underlining versioning mechanism (and version IDs map to different data types). It's now optional parameter.